joomla/joomla-cms
# [4.3] Fix Media Field with Windows directory separator in value

Pull Request for Issue #39203.

### Summary of Changes
Sometime, on a window system, value stored for a media form field has this format `images\headers\banner.jpg` and it will throw warning and there will be notice like below when you edit the record contains that field:

`Notice: Undefined offset: 0 in ...\libraries\src\Form\Field\MediaField.php on line 294`

This PR just fixes that notice.

### Testing Instructions
If you want to fully re-procedure the issue, you need to have Joomla setup on windows and follow instructions at https://github.com/joomla/joomla-cms/issues/39203#issuecomment-1312715797 to see the issue. However, there is a simple way to re-procedure it :

1. Add the line of code below after this line https://github.com/joomla/joomla-cms/blob/4.2-dev/libraries/src/Form/Field/MediaField.php#L263  . That will fake name of the image file in window directory separator format
```php
$this->value = 'images\headers\banner.jpg';
```

### Actual result BEFORE applying this Pull Request
You see notice like below:

`Notice: Undefined offset: 0 in ...\libraries\src\Form\Field\MediaField.php on line 294`

### Expected result AFTER applying this Pull Request

No notice anymore.
